packages/react-composites/src/composites/OneToOneCall/styles/ScreenSharePopup.styles.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getTheme, IButtonStyles, IModalStyleProps, IModalStyles, IStyleFunctionOrObject } from '@fluentui/react';
+import { CSSProperties } from 'react';
+
+const theme = getTheme();
+const palette = theme.palette;
+
+export const screenSharePopupModalStyles: IStyleFunctionOrObject<IModalStyleProps, IModalStyles> = {
+  root: { width: '100%', height: '100%' },
+  main: {
+    boxShadow: theme.effects.elevation8,
+    borderRadius: theme.effects.roundedCorner4,
+    background: palette.neutralLighterAlt
```

Comment:
Does this look okay in dark theme? I'm wondering because getTheme does not pick up the FluentThemeProvider's theme.

Reply:
In ScreenSharePopup.tsx, you can go:
`
import { concatStyleSets } from '@fluentui/react';
import { useTheme } from '@fluentui/react-theme-provider';

const theme = useTheme();
const modalStyles = concatStyles(screenSharePopupModalStyles, { root: { background: theme.palette.neutralLighterAlt } });
...
return (
<Modal isOpen={true} isModeless={true} dragOptions={DRAG_OPTIONS} styles={screenSharePopupModalStyles}>
...
`
or you can parameterize screenSharePopupModalStyles to take in theme from useTheme as an arg

Reply:
Thanks for the advice Miguel, I applied it to the PR. Here's how it looks in dark theme now:

![image](https://user-images.githubusercontent.com/3343009/120696609-9a07b400-c461-11eb-9d07-a0b15d0a347c.png)